Automattic/jetpack
# IDC: add spinner for the "Stay in safe mode" button

#### Changes proposed in this Pull Request:
- add spinner for when the "Stay in safe mode" button gets clicked
- rename CSS classes to follow the guidelines
- change the border color for "Successfully Migrated" modal

#### Jetpack product discussion
p9dueE-3pz-p2#comment-6407

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

1. Connect Jetpack, go to Jetpack Debug and activate the IDC Simulator.
2. Go to "Tools -> Connection Manager", confirm that you don't see the IDC screen.
3. Go to "Jetpack Debug -> IDC Simulator", enable "IDC Simulation" and enter a "Spoof URL".
4. Edit a post and save to make the site sync. Wait a few seconds.
5. Reload the "Tools -> Connection Manager". Confirm that you see the new RNA IDC screen. The old IDC banner will also appear, no worries, we'll hide it later.
6. Click "Stay in Safe Mode". The line should get replaced with the spinner and the text "Finishing setting up Safe mode…".
7. After the API request has succeeded, the page should reload, both IDC banners should disappear.
8. "Jetpack Safe Mode" button should appear in the top right corner, click it. The IDC banner should appear again.

There's also a number of CSS classes renamed, but I tested all the IDC scenarios, so probably no need to retest them all.
We'll test everything again later on, before undergoing the design review.
